### DISCUSSION TITLE
Navigating around reverted commit

### DISCUSSION THREAD

**Discussion by lowriderdog372** [community/community#146682] [New to GitHub]
I just finished a VS2022 project and need to upload my final commit to github.  It appears I reverted from the last commit and completed the project.  Now, I cannot push the final commit with the following error:

![Screenshot](https://github.com/user-attachments/assets/2f1846fb-5bcc-49ac-9f10-6ffb025a1f26)

The only problem is that there is no information in the output window.  I tried to create a new fork to work around the issue but I get the same error.  I am afraid to 'pull before push' as I don't want anything in my final to be changed.

How do I work around this?

**Comment by nishanmudalige** [comment]
Have you tried to force a push:

git push --force origin
